### DISCUSSION TITLE
Clicking on Review Changes not working

### DISCUSSION THREAD

**Discussion by aandi-ads** [community/community#78333] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When we navigate to PR and click on the review changes drop down is not working. Refresh the page and try to click, it's started working.

![image](https://github.com/community/community/assets/89501034/70606bd8-6761-41ba-9297-f064af6445ab)

**Comment by erickdc7** [comment]
Hello,

If you're experiencing issues with the "Review Changes" dropdown not working on a GitHub pull request (PR), it could be due to a temporary glitch or a browser-related problem. Here are some steps you can take to troubleshoot the issue:

1. **Refresh the Page**: As you mentioned, refreshing the page seems to resolve the issue temporarily. This could indicate a temporary loading issue.
2. **Clear Browser Cache**: Sometimes, clearing the browser cache can resolve issues with web applications not functioning correctly.
3. **Try a Different Browser**: To rule out browser-specific issues, try accessing the PR using a different web browser.
4. **Disable Browser Extensions**: Browser extensions can sometimes interfere with web applications. Try disabling any extensions that might affect GitHub's functionality.
5. **Check GitHub Status**: Visit the [GitHub Status page](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) to see if there are any ongoing issues that might be affecting the site's functionality.
6. **Contact GitHub Support**: If the problem persists, consider reaching out to GitHub Support for assistance.

For more detailed information on reviewing changes in a pull request, you can refer to the [GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request).

I hope this helps you resolve the issue.
